source/github-helpers/get-pr-info.ts

```diff
@@ -0,0 +1,24 @@
+import * as api from './api';
+import {getConversationNumber} from '.';
+
+interface PullRequestInfo {
+	// TODO: Probably can be used for #3863 and #4679
```

Comment:
Not sure what you mean by this comment
https://github.com/refined-github/refined-github/blob/ebfb5f62b9f92a677402ffe9d61d9c1eee27824c/source/features/restore-file.tsx#L18-L27

We are already using it